libs/aws/langchain_aws/rerank/rerank.py

```diff
@@ -0,0 +1,142 @@
+from copy import deepcopy
+from typing import Any, Dict, List, Optional, Sequence, Union
+
+import boto3
+from langchain_core.callbacks.manager import Callbacks
+from langchain_core.documents import BaseDocumentCompressor, Document
+from langchain_core.utils import from_env
+from pydantic import ConfigDict, Field, model_validator
+from typing_extensions import Self
+
+
+class BedrockRerank(BaseDocumentCompressor):
+    """Document compressor that uses AWS Bedrock Rerank API."""
+
+    client: Any = None
+    """Bedrock client to use for compressing documents."""
+    top_n: Optional[int] = 3
+    """Number of documents to return."""
+    model_id: Optional[str] = "amazon.rerank-v1:0"
```

Comment:
Update this to model_arn.
```suggestion
    model_arn: str
```

Reply:
Solved on f45340b485b3512503c34667c557c516f74fcfd9